controllers/article-controllers.js

```diff
@@ -19,7 +19,7 @@ exports.getArticleById = (req, res, next) => {
 };
 
 exports.getAllArticles = (req, res, next) => {
-  returnArticles()
+  returnArticles(req.query)
```

Comment:
I would prefer to see this de-structured in the controller file and passed down as exact values for the model to access.